src/issues/util.ts

```diff
@@ -584,7 +584,7 @@ export function getUpstreamOrigin(upstream: Remote, resultHost: string = 'github
 			fetchUrl = fetchUrl.substr('ssh://'.length);
 		}
 		// upstream's origin by ssh
-		if (fetchUrl.startsWith('git@') && !fetchUrl.startsWith('git@github.com')) {
+		if ((fetchUrl.startsWith('git@') || fetchUrl.includes('@git')) && !fetchUrl.startsWith('git@github.com')) {

```

Comment:
Issue was that we would only match `git@git.musta.ch:organization/repo.git`, not `org-4@git.musta.ch:organization/repo.git`

Open to other suggestions as well - e.g. matching `.endsWith('.git')` instead